### DISCUSSION TITLE
Expanded Instrument to BarChart Code List

### DISCUSSION THREAD

**Discussion by BorderCollieLover** [bug-or-feature/bc-utils#19] [General]
First I'd like to thank @bug-or-feature for sharing this project, which certainly helped save me a lot of time. 

Andy also provided an extended dictionary mapping PST instruments to BarChart code/exchange pairs in this repository. 

I tried to expand Andy's existing dictionary to include more instruments from PST. The results are in the attached spreadsheet. I wrote a brief explanation which I hope is enough for anyone wishing to make use of the results therein. 

The ultimate data validation would be to check downloaded BarChart data against PST data (for instruments shipped with adjusted and multiple price data) or against IB data (for instruments that didn't ship with adjusted and multiple price data), which is tedious and time-consuming (particularly for BarChart data with its 250-set limits). 

For this reason, I didn't expand CONTRACT_MAP and create a pull request. I also believe people might make use of this file in different ways depending on their specific setup. 

Hope this can be helpful, and appreciate if you let me know any errors you might come across in this file. 

[ExpandedInstrumentList.xlsx](https://github.com/user-attachments/files/16364336/ExpandedInstrumentList.xlsx)

**Comment by bug-or-feature** [comment]
@BorderCollieLover thanks

**Comment by littlebigfis** [comment]
That's a big timesaver thank you! @BorderCollieLover
